bin/node/src/chain_spec.rs

```diff
@@ -226,38 +123,36 @@ pub fn development_config(chain_params: ChainParams) -> Result<ChainSpec, String
     ))
 }
 
-pub fn testnet1_config(chain_params: ChainParams) -> Result<ChainSpec, String> {
+pub fn config(
+    chain_params: ChainParams,
+    authorities: Vec<AuthorityKeys>,
+    chain_id: &str,
+) -> Result<ChainSpec, String> {
     let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
 
-    let n_members = std::fs::read_to_string("/tmp/n_members")
-        .expect("Committee size is not specified")
-        .trim()
-        .parse::<usize>()
-        .expect("Wrong committee size");
-
-    let authorities = read_keys(n_members);
-
     let sudo_account: AccountId = hex![
         // 5F4SvwaUEQubiqkPF8YnRfcN77cLsT2DfG4vFeQmSXNjR7hD
         "848274306fea52dc528eabc8e14e6ae78ea275bc4247a5d6e2882ac8e948fe68"
     ]
     .into();
 
-    // Give money to the faucet account.
+    // Give money to the faucet account
     let faucet: AccountId = FAUCET_HASH.into();
     let rich_accounts = vec![faucet, sudo_account.clone()];
+
     Ok(ChainSpec::from_genesis(
         // Name
-        "Aleph Zero",
+        "AlephZero",
```

Comment:
Antek was insisting there should be a space there.

Reply:
I guess we want it to be a command line argument as well? We want to provide different name for Testnet and for Mainnet.

Reply:
ok, this and the token symbol will now come form cli (with some sensible defaults)